addons/l10n_fr_fec/wizard/account_fr_fec.py

```diff
@@ -22,6 +22,7 @@ class AccountFrFec(models.TransientModel):
     fec_data = fields.Binary('FEC File', readonly=True)
     filename = fields.Char(string='Filename', size=256, readonly=True)
     test_file = fields.Boolean()
+    exclude_zero = fields.Boolean(string="Exclude lines at 0", default=False)
```

Comment:
It is not possible to add stored field in a stable version

Reply:
Just something like
```bash
TOPICK=`git log -1 --pretty="%h"`
git fetch origin master
git reset --hard origin/master
git cherry-pick $TOPICK
git push --force-with-lease
```

Then retarget the PR

Reply:
Thanks a lot <3